Multitouch Support/Native/VoodooI2CNativeEngine.cpp

```diff
@@ -92,6 +91,14 @@ MultitouchReturn VoodooI2CNativeEngine::handleInterruptReport(VoodooI2CMultitouc
         
     super::messageClient(kIOMessageVoodooInputMessage, voodooInputInstance, &message, sizeof(VoodooInputEvent));
     
+    if (!isForceClickEnabled()) {
+        RelativePointerEvent event;
+        event.dx = 0;
+        event.dy = 0;
+        event.buttons = firstTransducer->physical_button.value();
+        event.timestamp = timestamp;
+        super::messageClient(kIOMessageVoodooTrackpointRelativePointer, voodooInputInstance, &event, sizeof(event));
+    }
```

Comment:
Do we need to send an event on *each* report rather than just reports with a button involved?

Reply:
It's sufficient to send only on state change (both pressing and unpressing).

Reply:
In that case, would you want to track the state changes instead, or just keep sending it on every report?